src/components/WidgetList/WidgetList.js

```diff
@@ -13,10 +13,10 @@ const WidgetList = props =>
         <Number title="Week" data={props.weekNumber} />
       </div>
       <div className="col-xs-12 col-sm-3">
-        <Number title="Load Time" data={props.newRelicLoadTime} />
+        <Number title="Load Time" data={props.newRelicLoadTime} riseIsGood={false} />
```

Comment:
You know what? :) It's better to name it `riseIsBad`. This ways it will default to "false", and we can specify it in the jsx simply as `riseIsGood`, e.g.
```<Number data={...} riseIsBad /> ```